components/MyFooter.js

```diff
@@ -11,7 +12,7 @@ function MyFooter() {
                 width: "100%",
             }}
         >
-            <a href="https://lfortran.org/">LFortran</a> by <a href="https://lcompilers.org/">LCompilers</a>
+            <a href="https://lfortran.org/">LFortran</a> ({commit.id}) by <a href="https://lcompilers.org/">LCompilers</a>
```

Comment:
How about this:
```suggestion
            <a href="https://lfortran.org/">LFortran</a> (<a href="https://github.com/lfortran/lfortran/commit/{commit.id}">{commit.id}</a>) by <a href="https://lcompilers.org/">LCompilers</a>
```
I tested that the shorter hash still works in the url. I didn't test the PR with this change.